shepherd

```diff
@@ -97,6 +97,11 @@ update_services() {
     else
       logger "Trying to update service $name with image $image" "true"
 
+      tasks_num=$(docker service ls | grep "$name" | awk '{print $4}')
```

Comment:
- I like `num_tasks` better as a variable name (seems more concise to me)
- `docker service ls -f "name=$name"` seems more safe as a filter